src/lib/linting.js

```diff
@@ -6,24 +6,24 @@
  * It receives the parsed AST and rules from ../svglint.js, and then runs each
  *   rule and gathers the results.
  */
-import { EventEmitter } from "events";
-import path from "path";
-import process from "process";
-import * as cheerio from "cheerio";
-import * as parse from "./parse.js";
-import Reporter from "./reporter.js";
-import Logger from "./logger.js";
+import {EventEmitter} from 'node:events';
```

Comment:
You need to update the Rollup configuration to define `node:*` imports as externals. Inside the *rollup.config.js* file prefix all Node.js imports in `config.external` object with `node:` or the next warnings are raised:

```
node:events (imported by src/lib/linting.js, src/lib/logger.js, src/lib/reporter.js)
node:path (imported by src/lib/linting.js, src/lib/parse.js)
node:process (imported by src/lib/linting.js, src/lib/parse.js, src/cli/util.js)
node:util (imported by src/lib/logger.js)
node:fs (imported by src/lib/parse.js)
```